devicon.json

```
@@ -8644,6 +8644,28 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "titaniumsdk",
+        "altnames": [],
+        "tags": [
+            "app",
+            "mobile",
+            "javascript",
+            "cross-platform",
+            "mobile",
+            "sdk"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#bd222b",
+        "aliases": []
```

Add alias. 
See [SVG-Versions#note](https://github.com/devicons/devicon/wiki/SVG-Versions#note) in our Wiki for more info :)
```suggestion
        "aliases": [
          {
            "base": "original",
            "alias": "plain"
          }
        ]
```